### ISSUE TITLE
eBMC LED Management :: Lamp Test :: Restore the Indicator LED status

### ISSUE THREAD

**Issue by vishwabmc** [ibm-openbmc/dev#2152]
After the lamp test is turned off, need to walk the asserted groups and then take the appropriate action for all the indicator LEDs.. 

At the end of this:  the state of Indicator LED would be **equal** to what it was prior to lamp test.

Tasks :  

- Modify LED Manager code to split inserting into asserted group and acting on the indicator LEDs as 2 distinct flows. see manager.cpp and group.cpp. 

    - Doing this will enable LED manager take the request and not actually act on the physical indicators if it sees LampTest object alive on the Dbus ( that is.. Update asserted group but do not driveLEDs if the lamptest is in progress )

- Have LampTest interface implementation call this [restoreStates] as part of putting [off]

**Comment by lxwinspur** [comment]
https://gerrit.openbmc-project.xyz/c/openbmc/phosphor-led-manager/+/36335

**Comment by mzipse** [comment]
@lxwinspur I see the above commit is abandoned.  Are you working a different commit now.  Any outlook on the work for this?

**Comment by lxwinspur** [comment]
@mzipse 
Sorry for my late reply.
Yeah, the above commit has been abandoned, and i have created a new commit, as follows:
https://gerrit.openbmc-project.xyz/c/openbmc/phosphor-led-manager/+/36623

Now, @vishwabmc needs to confirm the current task with other experts again, so we need to wait for Vishwa’s reply.
Thanks!

**Comment by mzipse** [comment]
Thank you @lxwinspur .

**Comment by mzipse** [comment]
@vishwabmc , George's commit above has been siting idle for about 6 weeks.  I think he needs feedback from you.  Can you take a look?

**Comment by vishwabmc** [comment]
> @vishwabmc , George's commit above has been siting idle for about 6 weeks. I think he needs feedback from you. Can you take a look?

Hi @mzipse not really.. This is supposed to have been WIP. But I did not realize it was all marked for Review.. Anyway, if not here on gerrit, I have been working with George regularly on this on IRC. Since the LampTest was taking different shapes from the interface standpoint, I did not want to keep changing this and hence stalled this intentionally with mutual agreement with George.
